src/components/Office/ShipmentServiceItemsTable.module.scss

```
@@ -0,0 +1,27 @@
+@import '../../shared/styles/basics';
+@import '../../shared/styles/mixins';
+@import '../../shared/styles/colors';
+@import '../../shared/styles/custom';
+
+.serviceItemsTable {
+  thead th {
+    @include u-bg('transparent');
+    @include u-padding-left(1.5);
+    border-top: none;
+  }
+  td {
+    @include u-bg('transparent');
+  }
+
+  caption {
+    text-align: left;
+
+    h4 {
```

The inherited margins from the h4 are causing some of the spacing to be off. I'd recommend resetting here with a `@include u-margin(0);` applied to our h4, and then add 16px of space to the bottom by adding a `@include u-margin-bottom(2);` after that.

```suggestion
    h4 {
    @include u-margin(0);
    @include u-margin-bottom(2);
```

---

Yep that's an improvement, good catch.